coralproject/talk
# Admin asset list

## What does this PR do?

Creates a list where admins can review assets with comment streams.

## How do I test this PR?

1. Create a few streams by visiting `http://localhost:300/assets/some-random-string`
2. Visit the admin panel.
3. Click "Streams" tab.
4. You should see a list of streams.
5. Sort, filter and search should work as expected.
6. Pagination should work as expected if more than 25 streams are present.

## Note
There is an unrelated bug causing publication dates to be incorrect.
